_config/redirects.yml

```diff
@@ -85,6 +85,9 @@ search/downloads:                          /search/releases/
 search/downloads.html:                     /search/releases/
 subprojects/search/docs:                   /search/documentation/
 subprojects/search/docs.html:              /search/documentation/
+search/documentation/migrate/6.0/:          https://docs.jboss.org/hibernate/search/6.0/migration/html_single/
+search/documentation/migrate/6.0/index.html: https://docs.jboss.org/hibernate/search/6.0/migration/html_single/
+search/documentation/migrate/6.0:          https://docs.jboss.org/hibernate/search/6.0/migration/html_single/
```

Comment:
I'm asking for curiosity, since there is probably a reason for. Why are the paths absolute?